### PR TITLE
feat(typegen): handle scalar types

### PR DESCRIPTION
**Pull request by zlesnr**

### CODE REVIEW

File: pkg/nerdgraph/typegen.go [hunk 1]
```diff
@@ -25,7 +29,7 @@ func ResolveSchemaTypes(schema Schema, typeNames []string) (map[string]string, e
 }
 
 func handleEnumType(schema Schema, t SchemaType) map[string]string {
-	types := make(map[string]string)
+	typeKeeer := make(map[string]string)
```
Comment (ctrombley) [inline pkg/nerdgraph/typegen.go:32]: typo?

File: pkg/nerdgraph/typegen.go [hunk 2]
```diff
@@ -7,6 +7,10 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+var (
+	types = make(map[string]string)
```
Comment (jthurman42) [inline pkg/nerdgraph/typegen.go:11]: Does this need to be package global?

Comment (zlesnr) [reply pkg/nerdgraph/typegen.go:11]: I'm trying to figure that out now.

File: pkg/nerdgraph/typegen.go (hunk 1 above)
Comment (zlesnr) [reply pkg/nerdgraph/typegen.go]: Yes.

**Comment by zlesnr** [comment]
This is not currently generating the enums.  This should be fixed before merge.

File: pkg/nerdgraph/typegen.go (hunk 2 above)
Comment (zlesnr) [reply pkg/nerdgraph/typegen.go:11]: Currently, yes.  The recursion makes it such that the global is updated when a new type is seen, so we don't end up with duplicates, or multiple fetches for the same time.

**Comment by zlesnr** [comment]
The code is now able to compile with the generated types.

**Pull request merged**